pkg/controller/jobs/rayjob/rayjob_controller_test.go

```diff
@@ -123,6 +123,101 @@ func TestPodSets(t *testing.T) {
 	}
 }
 
+func TestPodSets_WithNumOfHosts(t *testing.T) {
+	job := testingrayutil.MakeJob("job", "ns").
```

Comment:
```suggestion
func TestPodSets(t *testing.T) {
	testCases := map[string]struct{
```

@andrewsykim Could you align with the https://github.com/andrewsykim/kueue/blob/e98217bf21ee4741eab3ff8ff81107f75e97eeb3/pkg/controller/jobs/rayjob/rayjob_controller_test.go#L35 in the main branch so that we can avoid the conflicts in the future commit?

Reply:
done